tracing/src/main/java/com/palantir/remoting2/tracing/Tracer.java

```diff
@@ -51,14 +51,23 @@ protected Trace initialValue() {
     private static TraceSampler sampler = AlwaysSampler.INSTANCE;
 
     /**
+     * Creates a new trace, but does not set it as the current trace. The new trace is {@link
+     * Trace#isObservable observable} iff the given flag is true, or, iff {@code isObservable} is absent,
+     * if the {@link #setSampler configured sampler} returns true.
+     */
+    public static Trace createTrace(Optional<Boolean> isObservable, String traceId) {
```

Comment:
Yes, and it could be package private, or per my 3rd comment, be removed in favor of a clearer purpose "clearTrace" method.